mos-platform/commodore/cbm_k_acptr.s

```diff
@@ -0,0 +1,10 @@
+.text
+
+#include <cbm_kernal.inc>

```

Comment:
This is actually a comment, so cbm_kernal.inc isn't included.

The reason this still works is that the assembler treats ACPTR as an undefined external symbol, and the linker is able to fix it up later to point to the actual ACPTR. This will break if the user defines a symbol named ACPTR, which the C standard requires libc to accept.

You can fix this by using the assembly language include directive: `.include "cbm_kernal.inc"`. Alternatively, you can let the linker handle this, but use the reserved symbol `__ACPTR`, which users cannot legally define.

The same logic applies to most of the files in this PR.